org/ystia/docker/containers/generic/playbooks/docker_container_tasks.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Ystia Forge
+# Copyright (C) 2020 Bull S. A. S. - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+#
+
+- name: setup docker env
+  set_fact:
+    DOCKER_ENV: "{{ENV_VARS}}"
+  when: ENV_VARS | length > 0
+
+- name: setup docker exposed ports
+  set_fact:
+    DOCKER_EXP_PORTS: "{{EXPOSED_PORTS}}"
+  when: EXPOSED_PORTS | length > 0
+
+- name: setup docker published ports 
+  set_fact:
+    DOCKER_PUB_PORT: "{{PUBLISHED_PORTS}}"
+  when: PUBLISHED_PORTS | length > 0
+
+- name: setup docker volumes 
+  set_fact:
+    DOCKER_VOLUMES: "{{VOLUMES}}"
+  when: VOLUMES | length > 0
+
+- name: debug env
```

Comment:
Env vars may expose some secrets I think we should remove this

Reply:
ok, removed